### ISSUE TITLE
centos 8 packages missing

### ISSUE THREAD

**Issue by bby-bishopclark** [zeek/zeek#2406]
https://download.opensuse.org/repositories/security:/zeek/CentOS_8_Stream/x86_64/ is empty.  Is this expected?

Can we also begin talking about EL9 too ?

**Comment by ckreibich** [comment]
Yes thank you, we're aware of it. It's an issue with the openSUSE build service — see [this thread](https://community.zeek.org/t/centos-8-stream-installation/6637) for more information. Feel free to chime in there.

As for EL9, duly noted. It's just a matter of finding the time. If you're up for building Zeek yourself, it should work fine — we've been covering it in CI for a while now.

**Comment by bby-bishopclark** [comment]
I'd like this to remain open pending a fix upstream, if there's any way to request updates from SuSE.  Otherwise, do we have a setup on COPR to build where we know it works for centos/rocky/navylinux 8?  Will this be a problem with official releases on Enterprise Linux for 9 and the foreseeable future?
